### DISCUSSION TITLE
[Feature] Don't tag the same person twice when face recognized twice in one picture

### DISCUSSION THREAD

**Discussion by dekiesel** [immich-app/immich#6598] [Feature Request]
### The feature

Apparently my daughter and son look exactly alike to the algo that does face recognition. It would be great if immich could choose the _n_ most likely people when recognizing the same person _n_-times in one picture.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by eygraber** [comment]
That wouldn't work so well for collages, photos with mirrors, etc...

**Reply by dekiesel** [reply]
I don't know how often people take pictures of mirrors or collages, but imo it's still an improvement because you can manually change the recognized person to the right one. 

That's not possible at the moment when two different faces are tagged as the same..

**Reply by kingvuke** [reply]
For me it possible. At the beginning you see the person once, if you click on the pencil, you will see the person several times and can remove/replace them selectively.

**Reply by dekiesel** [reply]
Oh...well... that works... I only have group pictures so far and with all the people on them I didn't realize that the same person showed up twice when you click on edit.

Thanks for letting me know!

**Comment by rayzorben** [comment]
This doesn't work so great when you have non-identical twins and both are tagged as the same person in over 100K photos :(
